project.mml

```diff
@@ -100,7 +100,6 @@ Layer:
             WHERE (landuse IN ('forest', 'farmland', 'residential', 'commercial', 'retail', 'industrial', 'meadow', 'vineyard', 'orchard')
               OR "natural" IN ('wood', 'wetland', 'mud', 'sand', 'scree', 'shingle', 'bare_rock', 'heath', 'grassland', 'scrub'))
               AND way_area > 0.01*!pixel_width!::real*!pixel_height!::real
-              AND building IS NULL
```

Comment:
Did you check performance implications of this?

Reply:
No, I just checked that nothing is broken. That's why I claim only code cleaning, not a performance boost.

Reply:
This change will probably have a performance regression. There is an important index on the polygon table ([the "nobuilding" index](https://github.com/kocio-pl/openstreetmap-carto/blob/41c12c84e936aebece02351bae36c2b23b524522/indexes.yml#L20-L21)) and the `building is null` part of the query is there in order to use this index and speed up queries. 

It shouldn't be removed without careful testing.

Reply:
Yes, this change needs testing. If it hasn't been tested, we should revert this line.

Reply:
Could you make such test?

Reply:
Not in the near future. I think this line should be reverted as it's outside the scope of code cleanups to the landcover SQL.

Reply:
OK, PR is ready - #3622.